frontend/src/components/activity/content/LAThematicArea.vue

```diff
@@ -1,26 +1,27 @@
 <template>
   <card-content-node v-bind="$props">
-    <v-list three-line class="mx-n4">
-      <v-list-item-group>
+    <v-list three-line class="mx-n4 ec-lathematicarea">
+      <v-list-item-group multiple :value="model">
```

Comment:
Is v-list-item-group really the correct component to use here? When navigating via keyboard, I can now focus either the checkbox or the whole list item. If I try to "activate" the list item via spacebar, it plays an animation but has no effect on the checkbox. Could you either make sure that only the checkboxes are focusable, or at least make the list item toggle the checkbox when "clicked" via keyboard?